### PR TITLE
Update Python settings references

### PR DESCRIPTION
**Pull request by luabud**
For https://github.com/microsoft/vscode-python/issues/20267

### CODE REVIEW

File: docs/python/settings-reference.md [hunk 1]
```diff
@@ -68,6 +69,8 @@ The language server settings apply when `python.languageServer` is `Pylance` or
 | inlayHints.functionReturnTypes | false |  Whether to display inlay hints for function return types.  Accepted values are `true` or `false`. |
 | inlayHints.pytestParameters | false | Whether to display inlay hints for pytest fixture argument types. Accepted values are `true` or `false`. |
 | diagnosticSeverityOverrides | {} | Allows a user to override the severity levels for individual diagnostics. <br> For each rule, the available severity levels are `error` (red squiggle), `warning` (yellow squiggle), `information` (blue squiggle), and `none` (rule disabled). <br> For information about the keys to use for the diagnostic severity rules, see the **Diagnostic severity rules** section below. |
+| fixAll | `[]` | A list of code actions to run when running the **Fix All** command. <br> Accepted values in this list are `source.unusedImports` to remove all unused imports in the open file, and `source.convertImportFormat` to convert the imports according to the `python.analysis.importFormat` setting. |
```
Comment (luabud) [inline docs/python/settings-reference.md:72]: cc @heejaechang @rchiodo @debonte @bschnurr to make sure this sounds right :)

Comment (heejaechang) [reply docs/python/settings-reference.md:72]: `fix all` can be issued from "fix all" command

image

or "editor.codeActionsOnSave"

image

and yes, currently we only support 2 source actions you listed above. but the list can grow. so might be better to leave the description as open ends?

we have [README](https://github.com/microsoft/pyrx/blob/main/packages/vscode-pylance/README.md) that lists accepted value

image

**Review by cwebster-99 — DISMISSED** [review]

**Review by brettcannon — APPROVED** [review]

**Review by meththot — APPROVED** [review]

**Review by gregvanl — APPROVED** [review]
Thanks!

**Review by meththot — COMMENTED** [review]
Thanks

**Pull request merged**